src/project/pages/header-menu.page.js

```diff
@@ -0,0 +1,19 @@
+import { Page } from 'playwright';
+import { expect } from 'playwright/test';
+
+export class HeaderMenu {
+
+    constructor(page) {
+        this.page = page;
+        this.signInBtn = this.page.locator('.header_signin');
+        this.myProfileBtn = this.page.locator('#userNavDropdown');
+    }
+
+    async clickOnSignInBtn() {
+        await this.signInBtn.click();
+    }
+
+    async isMyProfileBtnVisible(){
+        await expect(this.myProfileBtn).toBeVisible();
+    }
```

Comment:
method called with prefixes `is, has` etc. should return boolean value

this methods returns nothing